src/themes/teams/components/Accordion/accordionStyles.ts

```diff
@@ -0,0 +1,11 @@
+import { IComponentPartStylesInput, ICSSInJSStyle } from '../../../../../types/theme'
+
+const accordionStyles: IComponentPartStylesInput = {
+  root: (): ICSSInJSStyle => ({
```

Comment:
Would it perhaps be easier to type the root object as `[key: string]: (): ICSSInJSStyle` (or whatever the dammed syntax is) so that every individual function doesn't have to declare types?

Reply:
Yes, did this however it doesn't provide completions inside the style object for index keys.  It will only do so if you type the return value.  Even that has many shortcomings.  Note the `const accordionStyles: IComponentPartStylesInput`, which is doing exactly what you are referring to.

Since this PR is being broken down into smaller bits, we'll save the typings for later.